packages/ui/src/components/NavigationBar/NavColumn.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { PropsWithChildren } from 'react';
+
+interface NavContainerProps {
+  readonly isNarrow?: boolean;
+  readonly className?: string;
+}
+
+export function NavColumn({ isNarrow, className, children }: PropsWithChildren<NavContainerProps>) {
+  const space = isNarrow ? 'space-x-[30px]' : 'space-x-14';
+  return <div className={`flex items-center ${space} ${className}`}>{children}</div>;
+}
```

Comment:
```suggestion
interface NavColumnProps {
  readonly isNarrow?: boolean;
  readonly className?: string;
}

export function NavColumn({ isNarrow, className, children }: PropsWithChildren<NavColumnProps>) {
  const space = isNarrow ? 'space-x-[30px]' : 'space-x-14';
  return <div className={`flex items-center ${space} ${className}`}>{children}</div>;
}
```
interface 이름이 NavColumnProps 가 되어야 할 것 같습니다. (의도가 있는거면 OK)